CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewPane.java

```diff
@@ -371,7 +373,9 @@ public NotesViewPane(final NotesViewController controller) {
     /**
      * Set the plugin reports that have executed on the current graph report.
      */
-    protected void setGraphReport(final Graph graph, final NotesViewController controller) {
+    protected void setGraphReport() {
```

Comment:
I think it might still be good to pass `graph` as a parameter and then allow the calling function to pass the active graph

Reply:
Alternatively could employ function overloading so you could have something like:
```
setGraphReport() {
  setGraphReport(GraphManager.getDefault().getActiveGraph());
}

setGraphReport(final Graph graph) {
 ...
}
```